config/manifests/metallb-frr-k8s-prometheus.yaml

```diff
@@ -1846,6 +1846,7 @@ rules:
   - pods
   verbs:
   - list
+  - get
```

Comment:
I am tempted to say this is not necessary as helm is passing but we didn't add the corresponding permission.
Also, we should've added it here https://github.com/metallb/metallb/blob/fe9434970028279a062a017c8d27c3867d23a18e/config/rbac/role.yaml#L152 and let the generation part do the rest. I am also surprised the CI didn't catch this :worried:

Reply:
ah this drove me mad (I even filed a wip pr to validate). You committed the changes in the kustomize bases in the first commit. I think it makes sense to move them to this commit too.

Reply:
The rbac permission of getting for pods was added by me manually at first.
After this modification, ci would check if manifests are  in sync by running `inv generatemanifests`. So I have to commit what `inv generatemanifests` modified (manifests in `config/manifests/`).
@fedepaol

Reply:
yes that makes sense. I am just arguing about having all of them in the same commit.
Also, you didn't add the rbac to the helm chart (but the helm lane is passing), so I'd check if that capability is really required (and if so, let's add it to the chart as well)

Reply:
Got you. I was mainly focused on the reconciler but did not pay much attention to the manifests and helm chart(thought these parts are covered by the `inv generatemanifests`).I'll try optimizing these parts.

Reply:
I'd give it a try without the rbac to see if it's really required.